### DISCUSSION TITLE
Bug found on website when sharing the links

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#425] [First-Timers] [answered]
If someone wants to get a green square for today on their GitHub profile, can you raise a bug on the repo https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io 

When sharing the website link, it shows Steve's face and Amadine's name 😂 . We should probably have it show the hero photo and our community name.

<img width="498" alt="Screenshot 2020-11-13 at 10 01 49" src="https://user-images.githubusercontent.com/624760/99060240-a4cb2c00-2597-11eb-952d-7dcf3ff3e62a.png">

**Comment by frankiefab100** [comment]
Okay, I'm working on that

**Reply by eddiejaoude** [reply]
Ok thanks, please add a link to the issue here also so people can find it easily 🤓

**Reply by eddiejaoude** [reply]
And also add a link to this discussion in the Issue too - makes it easy for people to see the history

**Reply by eddiejaoude** [reply]
Issue created https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues/116

**Comment by frankiefab100** [comment]
Alright

**Comment by jai-dewani** [comment, marked as answer]
I think can be marked as answered since [#134](https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/pull/134) got merged

**Reply by eddiejaoude** [reply]
yep 👍 thank you 🤓
